### PR TITLE
Fix: Inconsistency on Import from JSON button look

### LINKED ISSUE #20302: Import from JSON button design is jarringly different
**Describe the bug**
The button for 'Import from JSON' found on the wp_blocks list table is drastically different than it's sibling 'Add New' button.

**To reproduce**
1. Go to '/wp-admin/edit.php?post_type=wp_block'
2. See the two sibling buttons that are visually different in size.

**Expected behavior**
Both buttons should follow the same design convention for size.

**Screenshots**
Screen Shot 2020-02-18 at 1 32 49 PM

**Desktop (please complete the following information):**
 - OS: MacOS 10.15.3
 - Browser: Chrome 80.0.3987.87
 - Gutenberg: bundled version from WP 5.4-Beta2

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
## Description
Fixes: https://github.com/WordPress/gutenberg/issues/20302

This PR fixes the size and padding of the import from JSON button to make the button consistent with the Add New button.

## How has this been tested?
I verified the button still works (the UI can be accessed on .../wp-admin/edit.php?post_type=wp_block).

image

### CODE REVIEW

**Comment by mapk** [comment]
I just tested in Chrome and FF and now it looks too small.

**Chrome**

Screen Shot 2020-02-24 at 9 56 24 AM

**Firefox**

Screen Shot 2020-02-24 at 9 56 32 AM

The `is-small` class drops the font-size down to `11px` which is part of the problem I think. I'm still looking into this.

**Comment by mapk** [comment]
It appears there's a whole series of button heights going on here:

Screen Shot 2020-02-24 at 10 18 42 AM

**Comment by youknowriad** [comment]
Right now Gutenberg Buttons are 36 for regular buttons and 24 for small ones.

In Core it's more 30 for regular buttons and probably 24 too for small ones.

The "Add New" button though is an ad-hoc button in Core while "Import from JSON" is a regular Gutenberg button.

For the issue to be solved entirely,   we need:

 - Make sure Core buttons match Gutenberg buttons
 - Make sure the "Add new" is not an ad-hoc button.
 - Use the "same" variations between "Add New" and "Import from JSON"

This won't happen today though as the buttons refactoring in Core takes time.

Maybe we can just adapt the style of the "Import from JSON" in adhoc way now.

File: packages/list-reusable-blocks/src/style.scss [hunk 1]
```diff
@@ -3,7 +3,11 @@
 
 .list-reusable-blocks__container {
 	display: inline-flex;
-	padding: 9px 0 4px; // To match the H1
 	align-items: center;
-	vertical-align: top;
+	position: relative;
+	top: -3px;
+
+	.components-button {
```
Comment (ZebulanStanphill) [inline packages/list-reusable-blocks/src/style.scss]: Perhaps there should be a comment here noting that this is a temporary workaround until the core styles get updated?

**Comment by jorgefilipecosta** [comment]
Thank you for updates @mapk, things are looking good in my tests.

File: packages/list-reusable-blocks/src/style.scss (hunk 1 above)
Comment (jorgefilipecosta) [reply packages/list-reusable-blocks/src/style.scss]: Updated 👍

**Pull request merged**